eclair-core/src/test/scala/fr/acinq/eclair/db/SqlitePaymentsDbSpec.scala

```diff
@@ -138,10 +139,11 @@ class SqlitePaymentsDbSpec extends FunSuite {
     val s3 = s2.copy(id = UUID.randomUUID(), amountMsat = 88776655)
     db.addOutgoingPayment(s3)
 
-    db.updateOutgoingPayment(s3.id, FAILED)
+    db.updateOutgoingPayment(s3.id, FAILED, failures = Seq("one", "two", "three"))
```

Comment:
Can you also try to insert a record with the exactly same failure? It should work because the PRIMARY KEY for `sent_payments_failures` is not defined and it should use the `ROWID` field of sqlite, however if the primary key was defined as the compound of the two columns an attacker could return the same error twice causing a constraint violation when inserting the record.